R/activate.R

```diff
@@ -78,10 +78,9 @@ renv_activate_impl <- function(project,
     renv_imbue_self(project)
 
   # restart session if requested
-  if (restart && !is_testing())
-    return(renv_restart_request(project, reason = "renv activated"))
-
-  if (renv_rstudio_available())
+  if (restart)
```

Comment:
This change was needed as otherwise we don't run `load()` below, and so things like `the$project_path` are not set in tests.